recipes/harmony-py/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "harmony-py" %}
+{% set version = "0.4.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/harmony-py-{{ version }}.tar.gz
+  sha256: ae72c30e6fa767772de184f4a4fac7477577e9d536f6f36b16cc82af831fc00b
+  patches:
+    - 76.diff
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python >=3.6
+    - python-dateutil >=2.8.2,<2.9.dev0
+    - python-dotenv >=0.20.0,<0.21.dev0
+    - progressbar2 >=3.55.0,<3.56.dev0
+    - requests >=2.28,<3.dev0
+    - sphinxcontrib-napoleon >=0.7,<1.dev0
+    - curlify >=2.2.1,<2.3.dev0
```

```suggestion
    - python-dateutil >=2.8.2,<2.9.dev0
    - python-dotenv >=0.20.0,<0.21
    - progressbar2 >=3.55.0,<3.56
    - requests >=2.28,<3
    - sphinxcontrib-napoleon >=0.7,<1
    - curlify >=2.2.1,<2.3
```

There is no need to pin against `dev0` on conda-forge. (We don't have dev releases afaik.)

---

Good point @saraedum -- I just copy-pasted this from their requirements files that get loaded into the `setup.py`.